Kernel/Devices/HID/Management.h

```diff
@@ -51,6 +53,9 @@ class HIDManagement {
     void set_client(KeyboardClient* client);
     void set_maps(NonnullOwnPtr<KString> character_map_name, Keyboard::CharacterMapData const& character_map);
 
+    void attach_standalone_hid_mouse_device(HIDDevice&);
+    void detach_standalone_hid_mouse_device(HIDDevice&);
```

Comment:
Should probably generalize these names as it will allow attach/detach all sorts of HID devices.